eventhubs/data-plane/azure-eventhubs/src/main/java/com/microsoft/azure/eventhubs/impl/SessionHandler.java

```diff
@@ -61,6 +61,17 @@ public void onSessionLocalOpen(Event e) {
                 }
             }
 
+            if (reactorHandler == null) {
+                this.onRemoteSessionOpenError.accept(
+                    null,
+                    new EventHubException(
+                        false,
+                        String.format("OnSessionLocalOpen entityName[%s], reactorHandler: NULL POINTER exception.")
+                    )
+                );
+                return;
+            }
```

Comment:
Why was this added?

Reply:
Initially, the SpotBugs complaints reactorHandler has no null checking. So I add an null condition checking with onRemoteSessionOpenError(null, EventHubException) error assigned. It gives caller a NULL POINTER error message.  The reason to choose onRemoteSessionOpenError(null, EventHubException) is because I mimics the rest of usage of reactorHandler line 75 - 93

Reply:
I don't understand why any of this code needs to be added. @JamesBirdsall will hopefully provide guidance.

Reply:
@JamesBirdsall  do you think the changes here make sense or just return; will be sufficient?

Reply:
Looking at the reactor startup code in MessagingFactory, it looks like it's not possible to have a reactor without a ReactorHandler. We certainly haven't been getting reports of NPEs here. If it somehow is null, then the reactor is probably in a very bad state and going to crash somewhere else if we prevent the crash here. In any case, if we do decide to handle it, then we should also call session.close()